officers.json

```diff
@@ -55,6 +69,20 @@
     "term_end": "SPRING_2018",
     "graduating_semester": "FALL_2018"
   },
+  {
+    "name": "Mathew (매튜) Kim",
```

Comment:
Probably shouldn't have foreign characters for names, as they can look weird for some users

![image](https://user-images.githubusercontent.com/16447522/48438614-f858d400-e749-11e8-96a9-8859c587209f.png)

Reply:
Removed for now, we've said we'll fix this to support it later